docs/notebooks/plasticc_snia.ipynb

```
@@ -0,0 +1,234 @@
+    "from tdastro.math_nodes.np_random import NumpyRandomFunc\n",
+    "from tdastro.simulate import simulate_lightcurves\n",
+    "from tdastro.sources.sncomso_models import SncosmoWrapperModel\n",
+    "from tdastro.sources.snia_host import SNIaHost\n",
+    "from tdastro.utils.plotting import plot_lightcurves\n",
+    "\n",
+    "from tdastro import _TDASTRO_BASE_DATA_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0b1cad1",
+   "metadata": {},
+   "source": [
+    "# Load Data Files\n",
+    "\n",
+    "We start by loading the files we will need for running the simulation: the OpSim database and the passband information. Both of these live in the `data/` directory in the root directory. Note that nothing in this directory is saved to github, so the files might have to be downloaded initially."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da09419a-cf38-4d6f-b44c-e1110caaf541",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the OpSim data.\n",
+    "opsim_db = OpSim.from_db(_TDASTRO_BASE_DATA_DIR / \"opsim_db\" / \"baseline_v3.4_10yrs.db\")\n",
+    "t_min, t_max = opsim_db.time_bounds()\n",
+    "print(f\"Loaded OpSim with {len(opsim_db)} rows and times [{t_min}, {t_max}]\")\n",
+    "\n",
+    "# Load the passband data for the griz filters only.\n",
+    "passband_group = PassbandGroup(preset=\"LSST\", filters_to_load=[\"g\", \"r\", \"i\", \"z\"])\n",
+    "print(f\"Loaded Passbands: {passband_group}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab9d7cf5",
+   "metadata": {},
+   "source": [
+    "We don't care about any of the observations in the OpSim that are not in one of the filters of interest. We can use a combination of `PassbandGroup`'s `mask_by_filter()` to determine which observations match and `OpSim`'s `filter_rows()` to remove those rows that do not match. This is an optional step that will improve the speed at which the algorithm runs by removing unused data."
```

I think this is not optional because we currently need the opsim filters and the passband filters match each other otherwise it will error

---

Do you mean that it should throw an error instead? Or something else?

---

I meant the description was not accurate. Users will get an error if they don't do the filter, so it's not "optional"

---

Good call. I removed that sentence.